src/views/blocks/attendees/description.php

```diff
@@ -8,28 +8,36 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link https://m.tri.be/1amp Help article for RSVP & Ticket template files.
+ * @link    https://m.tri.be/1amp Help article for RSVP & Ticket template files.
  *
- * @since 4.9.2
- * @version 4.11.3
+ * @since   4.9.2
+ * @since   4.11.3
  *
+ * @version TBD Updated docblock vars and removed unnecessary second retrieving of Post ID.
+ *
+ * @var Tribe__Tickets__Editor__Template $this       Template object.
+ * @var int                              $post_id    [Global] The current Post ID to which tickets are attached.
+ * @var array                            $attributes [Global] Attendee block's attributes (such as Title above block).
+ * @var array                            $attendees  [Global] List of attendees with attendee data.
+ * @var array                            $attendees  List of attendees and their data.
  */
+
 $display_subtitle = $this->attr( 'displaySubtitle' );
 
 if ( is_bool( $display_subtitle ) && ! $display_subtitle ) {
 	return;
 }
 
-$post_id         = $this->get( 'post_id' );
 $attendees_total = count( $attendees );
-$message         = _n( 'One person is attending %2$s', '%d people are attending %s', $attendees_total, 'event-tickets' );
+
+$message = _n( 'One person is attending %2$s', '%d people are attending %s', $attendees_total, 'event-tickets' );
```

Comment:
* :warning: Mismatched placeholders is probably an error
* :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.
* :no_entry_sign: Missing singular placeholder, needed for some languages. See https://codex.wordpress.org/I18n_for_WordPress_Developers#Plurals
* :no_entry_sign: Multiple placeholders should be ordered. Expected '%1$d, %2$s', but got %d, %s.

<sub>hash: 793a74b0d4f2c3d596e6940643d78dc4</sub>